config/config.go

```diff
@@ -41,27 +39,28 @@ const (
 	SSLCertPathKey                      = "SSL_CERT"
 	SSLKeyPathKey                       = "SSL_KEY"
 	WireEncodingKey                     = "WIRE_ENCODING"
-	ReplicaGroupIDKey                   = "REPLICA_GROUP_ID" // TODO: read replica group id from IPFS metadata
-	PaymentExpirationThresholdBlocksKey = "PAYMENT_EXPIRATION_THRESHOLD_BLOCKS"
 	PaymentChannelStorageTypeKey        = "PAYMENT_CHANNEL_STORAGE_TYPE"
 	PaymentChannelStorageClientKey      = "PAYMENT_CHANNEL_STORAGE_CLIENT"
 	PaymentChannelStorageServerKey      = "PAYMENT_CHANNEL_STORAGE_SERVER"
 
 	defaultConfigJson string = `
 {
-	"auto_ssl_cache_dir": ".certs",
 	"auto_ssl_domain": "",
 	"blockchain_enabled": true,
 	"daemon_listening_port": 5000,
 	"daemon_type": "grpc",
+    "daemon_end_point": "http://localhost:8080",
 	"db_path": "snetd.db",
 	"ethereum_json_rpc_endpoint": "http://127.0.0.1:8545",
-    "IPFS_END_POINT": "http://localhost:5002/",
 	"hdwallet_index": 0,
 	"hdwallet_mnemonic": "",
+    "ipfs_end_point": "http://localhost:5002/", 
```

Comment:
Please pay attention on spaces :-) Probably I have used unusual for Golang formatting in this JSON, but I would prefer keep it consistent and fix in separate PR if it is bothering.